sdk/communication/azure-communication-chat/README.md

```diff
@@ -128,15 +128,14 @@ CreateChatThreadResult result = chatClient.createChatThread(createChatThreadOpti
 String chatThreadId = result.getChatThread().getId();
 ```
 
-#### Get a chat thread
+#### Get a chat thread properties
 
-The `getChatThread` method retrieves a thread from the service.
-`chatThreadId` is the unique ID of the chat thread.
+The `getChatThreadProperties` method retrieves a thread's properties from the service.
 
 <!-- embedme ./src/samples/java/com/azure/communication/chat/ReadmeSamples.java#L97-L98 -->
 ```Java
-String chatThreadId = "Id";
-ChatThread chatThread = chatClient.getChatThread(chatThreadId);
+ChatThreadClient chatThreadClient = chatClient.getChatThreadClient("Id");
```

Comment:
This is also a breaking change. This is not consistent with code. ChatThreadClient is built by the builder in code.

Reply:
are you saying we should remove the `getChatThreadClient` method?